src/ext/compatibility.h

```diff
@@ -39,6 +39,11 @@
 #define PHP7_UNUSED(...) UNUSED(__VA_ARGS__)
 #endif
 
+#if PHP_VERSION_ID < 70300
```

Comment:
This should apply only to 7.0-7.2. With the way it is written atm it will also apply to PHP 5.

Reply:
True, actually I had other defines for 5.6 but they didn't have desired effect in 5.x. I'll make this `if` as you've suggested.